docs/common/error-utilities.ts

```diff
@@ -487,4 +487,11 @@ const RENAMED_PAGES: Record<string, string> = {
 
   // After merging EAS environment variables guides
   '/eas/using-environment-variables/': '/eas/environment-variables/',
+
+  // After Expo Router Getting Started Guide
+  '/router/reference/authentication': '/router/advanced/authentication',
+  '/router/advanced/root-layout': '/router/basics/layout#root-layout',
+  '/router/reference/not-found': '/router/error-handling',
+  '/router/navigating-pages': '/router/basics/navigation',
+  '/router/create-pages': '/router/basics/core-concepts',
```

Comment:
```suggestion
  // After Expo Router Getting Started Guide
  '/router/reference/authentication/': '/router/advanced/authentication/',
  '/router/advanced/root-layout/': '/router/basics/layout#root-layout/',
  '/router/reference/not-found/': '/router/error-handling/',
  '/router/navigating-pages/': '/router/basics/navigation/',
  '/router/create-pages/': '/router/basics/core-concepts/',
```

Adding a `/` trailing slash is important for client-side redirects because in next.config.js, we dynamically set it but the redirect mechanism is not aware of it (afaik).

Adding a trailing a slash, they're working as expected. I've tested them locally.